plio/views.py

```diff
@@ -373,9 +373,15 @@ def metrics(self, request, uuid):
         # else fetch the object
         plio = self.get_object()
 
+        num_survey_questions = Question.objects.filter(item__plio=plio.id, survey=True)
```

Comment:
instead of redefining `questions` later, do this here itself:
```
questions = Question.objects.filter(item__plio=plio.id)
survey_questions = questions.filter(survey=True)